ui-v2/app/styles/core/typography.scss

```diff
@@ -109,6 +109,13 @@ pre code,
 }
 /**/
 
+/* composite row */
+%composite-row-header {
+  font-size: 1.125rem;
+  font-weight: $typo-weight-medium;
+}
```

Comment:
Thanks for moving this over here! Sorry, I just noticed something else, all of our other copy use a font variable. Typically this is a new value 😂 , luckily though we have one `900` space left in our fonts scale.

https://github.com/hashicorp/consul/blob/47832cc81d5bc3b0c8965b5d17d0b618e286ca4b/ui-v2/app/styles/base/typography/base-variables.scss#L4-L12

I think for the moment we add a `$typo-size-450: 1.125rem` here. I think this font scale was based on the fonts we were using before Structure, not Structure itself.

Actually just looking at the fonts in Structure it doesn't look to have this 1.125rem size either (I think that rem size equates to roughly 18px which doesn't exist in the Structure Text Styles). @opihana @jnwright should we add this size into Structure, or should we switch ours out for a font size from Structure?

@kaxcode I think in the meantime, let's go with adding the `$typo-size-450: 1.125rem` to the `typography/base-variables.css`.